pkg/porter/lifecycle.go

```diff
@@ -28,8 +39,12 @@ func (o *BundleLifecycleOpts) Validate(args []string, porter *Porter) error {
 	return nil
 }
 
+func (o BundleLifecycleOpts) GetBundleLifecycleOptions() BundleLifecycleOpts {
```

Comment:
Brilliant! I wouldn't have thought to implement this here, and would have put it on each of the actions but this is sneakier. 💯